rpc/ethereum/namespaces/eth/api.go

```diff
@@ -691,40 +691,22 @@ func (e *PublicAPI) GetTransactionByHash(hash common.Hash) (*rpctypes.RPCTransac
 // getTransactionByBlockAndIndex is the common code shared by `GetTransactionByBlockNumberAndIndex` and `GetTransactionByBlockHashAndIndex`.
 func (e *PublicAPI) getTransactionByBlockAndIndex(block *tmrpctypes.ResultBlock, idx hexutil.Uint) (*rpctypes.RPCTransaction, error) {
 	var msg *evmtypes.MsgEthereumTx
```

Comment:
no need for predefined variable anymore

Reply:
I've added the `/tx_search` solution back.